apps/sms/views/conversation/test/unit/attachment_test.js

```diff
@@ -145,62 +145,67 @@ suite('attachment_test.js', function() {
       this.sinon.spy(MimeMapper, 'ensureFilenameMatchesType');
     });
 
-    test('Open normal image attachment', function() {
+    test('Open normal image attachment', function(done) {
       var fileName = 'jpeg_image.jpg';
       var attachment = new Attachment(testImageBlob, {
         name: fileName
       });
       var typeSpy = MimeMapper.guessTypeFromFileProperties;
       var matchSpy = MimeMapper.ensureFilenameMatchesType;
-      attachment.view();
-      assert.ok(typeSpy.calledWith(fileName, 'image/jpeg'));
-      assert.ok(matchSpy.calledWith(fileName, typeSpy.returnValues[0]));
-      assert.equal(MockMozActivity.calls.length, 1);
+      attachment.view().then(() => {
+        assert.ok(typeSpy.calledWith(fileName, 'image/jpeg'));
+        assert.ok(matchSpy.calledWith(fileName, typeSpy.returnValues[0]));
+        assert.equal(MockMozActivity.calls.length, 1);
+      }).then(done, done);
     });
 
-    test('Filename has no extension', function() {
+    test('Filename has no extension', function(done) {
       var fileName = 'jpeg_image.jpg';
```

Comment:
It's not related to your patch, but this part seems wrong that filename should be `jpeg_image` instead of `jpeg_image.jpg`. Could you please fix this as since you're modifying the attachment unit test? Sorry for the inconvenience... :/